README.md

```diff
@@ -1,33 +1,26 @@
 # `zq` [![Tests][tests-img]][tests] [![GoDoc][doc-img]][doc]
 
-`zq` is a command-line tool for searching and analyzing logs,
-particularly [Zeek](https://www.zeek.org) logs.  If you are familiar with
-[`zeek-cut`](https://github.com/zeek/zeek-aux/tree/master/zeek-cut),
-you can think of `zq` as `zeek-cut` on steroids.
+The `zq` repository contains tools and components used to search, analyze
+, and store structured log data, including:
 
-`zq` is comprised of:
+* The [zq](cmd/zq/README.md) command line tool, for searching and analyzing log
+ files
+* The [zar](cmd/zar/README.md) command line tool, for working with log data
+ archives
+* The [zqd](cmd/zqd/README.md) daemon, which serves a REST-like API to manage
+ and query log archives, and is the backend for the [Brim](https://github.com/brimsec/brim)
+ application
```

Comment:
```suggestion
 application
 * The [zapi](cmd/zapi/README.md) command line tool, for interacting with the
API provided by zqd
```

When I saw zqd mentioned in my first review pass, I found myself wanting to see zapi on the list as well. That said, by the end of my review, I realized that exhaustively covering every single item in `cmd/` in this list risked distracting from the importance of the most important ones. That said, we know at least one member of our community now relies on zapi as part of their workflow, so this is validation that it's useful for more than just our internal dev/test purposes, so it seems worthwhile to call attention to it.